randovania/interface_common/options.py

```diff
@@ -385,14 +387,25 @@ def user_dir(self) -> Path:
 
     # Access to Direct fields
     @property
-    def last_changelog_displayed(self) -> StrictVersion:
-        return StrictVersion(self._last_changelog_displayed)
+    def last_changelog_displayed(self) -> version_lib.Version:
+        if randovania.is_dev_version():
+            version_str = self._last_changelog_displayed_dev
+        else:
+            version_str = self._last_changelog_displayed
+
+        return version_lib.parse_string(version_str)
```

Comment:
IMO, if version_str is an invalid version string, we should reset the field.
Just in the case that someone somehow gets an invalid field in there, that they aren't completely blocked from using the application.

Reply:
We really should be using `version_lib.parse_string` as the method for it in `_SERIALIZER_FOR_FIELD` because then it falls back to the popup saying "invalid field found. delete and continue?".
This error only happens when we cause an error, so I don't want to silently delete user data.